.gitignore

```diff
@@ -34,6 +34,12 @@ perf.data*
 jeprof.*
 .idea/
 .mypy_cache/
-*.csv
-*.pdf
+# *.csv
+# *.pdf
 tags
+
+bench-scripts/**/results
+circ-docker.tar.gz
```

Comment:
maybe a more general term for this?

Reply:
* `bench-scripts/**/results`: Instead of specifying the result directory, I just gitignored the PDFs and SVGs which the benchmark scripts produce.
* `circ-benchmark.tar.gz`: That line is a legacy; it is from artifact generating script, which is not part of the benchmark. Therefore, I have removed it.